docs/ql-documentation/learn-ql/cpp/introduce-libraries-cpp.rst

```diff
@@ -4,114 +4,516 @@ Introducing the C/C++ libraries
+| ``enum {`` **enum_constant1** ``,`` **enum_constant2** ... ``}``                                                                                                                                                                                                                                                                                                                                                                            | `EnumConstant <https://help.semmle.com/qldoc/cpp/semmle/code/cpp/Enum.qll/type.Enum$EnumConstant.html>`__                                                             |                                                                                                                                                                                                                                                                                   |
++---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``class A {`` **friend** ``void f (`` *int* ``);`` **friend** ``class X ; }; void f ( int x`* ``) {`` … ``} class X {`` … ``};``                                                                                                                                                                                                                                                                                                            | `FriendDecl <https://help.semmle.com/qldoc/cpp/semmle/code/cpp/FriendDecl.qll/type.FriendDecl$FriendDecl.html>`__                                                     |                                                                                                                                                                                                                                                                                   |
```

Comment:
Some backticks slipped through into the render. Search [GitHub's render of this file](https://github.com/zlaski-semmle/ql/blob/zlaski/cpp387/docs/ql-documentation/learn-ql/cpp/introduce-libraries-cpp.rst) to find the rest.